src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/actions/manual/ManualTriggerAction.java

```diff
@@ -236,20 +236,22 @@ public String getJsUrl(String jsName) {
      * Finds the highest and lowest code review vote for the provided patch set.
      *
      * @param res the patch set.
+     * @param patchSetNumber the patch set number.
      * @return the highest and lowest code review vote for the patch set.
      */
-    public HighLow getCodeReview(JSONObject res) {
-        return Approval.CODE_REVIEW.getApprovals(res);
+    public HighLow getCodeReview(JSONObject res, int patchSetNumber) {
```

Comment:
I get a bit nervous when I see public methods changing signature like this. Can the old signature be left and deprecated with some default value?